docs/documentation/index.html

```diff
@@ -1217,6 +1225,323 @@ <h5 class="m-documentation--configuration-side-title">UI configuration</h5>
+          GitLab API base URL <br>
+          <span class="tag">Default:</span> <code>https://gitlab.com/</code>
+        </dd>
+
+        <dt><code>MO_MONITORABLE_GITLAB_TOKEN</code> <code class="type">string</code> <span class="required">required</span>
+        </dt>
+        <dd>
+          Personal access token <br>
+          <a href="https://gitlab.com/profile/personal_access_tokens">
+            Create a personal access token for Monitoror
+          </a>
+        </dd>
+
+        <dt><code>MO_MONITORABLE_GITLAB_TIMEOUT</code> <code class="type">number</code></dt>
+        <dd>
+          Timeout in milliseconds before returning error <br>
+          <span class="tag">Default:</span> <code>5000</code>
+        </dd>
+      </dl>
+
+      <p class="configuration-variants-available">
+        <svg clip-rule="evenodd" fill-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2"
+             viewBox="0 0 510 510" xmlns="http://www.w3.org/2000/svg">
+          <path
+                  d="M130.122 165.707h101.614l83.236 211.052h133.126l-25.75 25.748c-6.332 6.332-6.332 16.628 0 22.96 6.331 6.331 16.624 6.331 22.955 0l64.941-64.941-64.941-64.941c-6.331-6.332-16.624-6.332-22.955 0-6.332 6.332-6.332 16.628 0 22.959l25.75 25.748H337.084L266.64 165.707h181.458l-25.75 25.748c-6.332 6.332-6.332 16.624 0 22.956 6.331 6.331 16.624 6.331 22.955 0l64.941-64.937-64.941-64.941c-6.331-6.331-16.624-6.331-22.955 0-6.332 6.332-6.332 16.624 0 22.956l25.748 25.748H130.122v32.47zm-32.47 0v-32.47H65.185v32.47h32.467zm-64.937 0v-32.47H.244v32.47h32.471z"
+                  fill="currentColor" fill-rule="nonzero"/>
+        </svg>
+        <a href="#configuration-variants">Configuration Variants</a> are available for GitLab
+      </p>
+
+      <pre class="example"><code>
+MO_MONITORABLE_GITLAB_URL="https://gitlab.com/"
+MO_MONITORABLE_GITLAB_TOKEN="thisisyourtoken"
+MO_MONITORABLE_GITLAB_TIMEOUT=5000
+      </code></pre>
+
+      <h4 id="tile-gitlab-issues">GITLAB-ISSUES</h4>
+
+      <p>
+        Show a counter of issues.
+      </p>
+
+      <h5 class="m-documentation--configuration-side-title">UI configuration</h5>
+
+      <dl>
+        <dt><code>projectId</code> <code class="type">number</code></dt>
+        <dd>
+          GitLab project ID (from project setting page)
+        </dd>
+
+        <dt><code>state</code> <code class="type">string</code></dt>
+        <dd>
+          GitLab issue state
+        </dd>
+
+        <dt><code>labels</code> <code class="type">[]string</code></dt>
+        <dd>
+          GitLab issue labels
+        </dd>
+
+        <dt><code>milestone</code> <code class="type">string</code></dt>
+        <dd>
+          GitLab issue milestone
+        </dd>
+
+        <dt><code>scope</code> <code class="type">string</code></dt>
+        <dd>
+          GitLab issue scope
+        </dd>
+
+        <dt><code>search</code> <code class="type">string</code></dt>
+        <dd>
+          Keyword to search in the title or description
```

Comment:
Only one keyword is allowed?

Reply:
Don't know, gitlab api say "string" :
https://docs.gitlab.com/ee/api/issues.html#list-project-issues

Reply:
In the example:

> issue+title+or+description

so, maybe we can support a space-separated list of keywords? :)

Reply:
it's already like that, if you use +, they are escaped and it does not work

Reply:
So change the docs to reflect that:

"Space-separated keywords to search in the title or description"